screens/AddHotspotScreen.tsx

```diff
@@ -0,0 +1,414 @@
+import { capitalize } from 'lodash';
+import React, { useState } from 'react';
+import {
+  Dimensions,
+  Image,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import {
+  Avatar,
+  Button,
+  Divider,
+  FAB,
+  TextInput,
+  useTheme,
+} from 'react-native-paper';
+import SelectDropdown from 'react-native-select-dropdown';
+import catLadyImage from '../assets/cat-lady2.png';
+import currentLocationIcon from '../assets/current-location.png';
+import mapPinIcon from '../assets/map-pin.png';
+import { Appbar } from '../components/Appbar';
+import { InputField } from '../components/InputField';
+import { HotspotStatus } from '../models/Hotspot';
+import { Location } from '../models/Location';
+
+('react-native-paper-dropdown');
```

Comment:
what is this good for?

Reply:
Not needed anymore. I've tried before a different library for dropdowns, and its import remained there. I removed it.